code/game/objects/buckling.dm

```diff
@@ -249,6 +249,11 @@
 	if(!Adjacent(user) || !Adjacent(target) || !isturf(user.loc) || user.incapacitated() || target.anchored)
 		return FALSE
 
+	if(iscarbon(user))
+		var/mob/living/carbon/carbon_user = user
+		if(carbon_user.num_hands <= 0)
```

Comment:
```suggestion
		if(carbon_user.usable_hands <= 0)
```
`usable_hands` counts disabled OR missing hands, while `num_hands` just counts hands regardless if they're disabled (broken) or not